src/api/src/attestation/amber/mod.rs

```diff
@@ -49,18 +50,18 @@ pub struct Amber {
 #[async_trait]
 impl Attest for Amber {
     async fn verify(&mut self, tee: Tee, _nonce: &str, attestation: &str) -> Result<String> {
-        if tee != Tee::Tdx {
-            bail!("Only implement for tdx now");
+        if tee != Tee::Tdx && tee != Tee::Sgx {
+            bail!("Amber: TEE {tee:?} is not supported.");
         }
         // get quote
         let attestation = serde_json::from_str::<Attestation>(attestation)
             .map_err(|e| anyhow!("Deserialize Attestation failed: {:?}", e))?;
-        let tdx_evidence = serde_json::from_str::<TdxEvidence>(&attestation.tee_evidence)
-            .map_err(|e| anyhow!("Deserialize TDX Evidence failed: {:?}", e))?;
+        let evidence = serde_json::from_str::<AmberTeeEvidence>(&attestation.tee_evidence)
```

Comment:
We'd better raise an `info` or `warn` here to tell the user that `eventlog` is ignored.

Reply:
With amber, `event_log` needs to be added as a separate field in the appraisal request but it's currently not done so there's no change in functionality with this PR.